### ISSUE TITLE
Zookeeper authentication

### ISSUE THREAD

**Issue by edevil** [disqus/menagerie#5]
Does this project support authentication?

**Comment by Fluxx** [comment]
That is a pretty open-ended question.  Could you elaborate?

**Comment by edevil** [comment]
Yeah, sorry. What I meant to ask was if Zookeeper authentication is supported. In the example I only see hosts being configured, not credentials.
